app/src/ledger_sdk_stub/nbgl_display.rs

```diff
@@ -138,5 +138,5 @@ unsafe extern "C" fn settings_callback(token: c_int, _index: u8, _page: c_int) {
 }
 
 unsafe extern "C" fn app_exit() {
-    os_sched_exit(1);
+    os_sched_exit(0);
```

Comment:
What value does Bitcoin use? 1 or 0?

Reply:
It's [using -1](https://github.com/LedgerHQ/ledger-secure-sdk/blob/a041acdc1e9479e3c0566d09f8ac5ae966c8274f/lib_standard_app/main.c#L40) in the Ledger C app, but the type is `unsigned char`. We cannot use `-1` in Rust, so I changed it to `u8::MAX`.

Reply:
Let's add a comment on why u8::MAX was used.